scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -9,6 +9,8 @@ This command will pick the default values specified in the [pom](https://github.
 
 Note: If you are planning to use GPU, please add `-Dmxnet.profile=linux-x86_64-gpu`
 
+Note: The Maven package is built with CUDA 9.2.
```

Comment:
Thanks Sam for bringing this up.
I think this is the correct location for this disclaimer : https://github.com/apache/incubator-mxnet/tree/master/scala-package#pre-built-maven-packages

Reply:
hmmm... I never looked at this page before. Maybe we should add a link and generalize the note more and say something like "Make sure you're environment is configured correctly according to..." and link to that page then?

Users following this demo arent necessarily going to read all the docs before running. We need to keep things self-containted and succinct.